### PR TITLE
GitLab.com support

### PR DESCRIPTION
**Pull request by Johannestegner**
Added a gitlab.js file which enables support for GitLab.com in the plugin.  
I tried to keep the file as close to the `github.js` file in style as possible.

Updated the config file and manifest to include the new script on build.

### CODE REVIEW

**Comment by kropp** [comment]
Thank you very much for your contribution!
I just want to point out that change to `permissions` in manifest.json will require all users to re-enable the extension. So this is something we should consider doing with a caution.

**Comment by Johannestegner** [comment]
@kropp happy to contribute!

I understand that the permission change is annoying, but I think(?) that it is required to access the GitLab domain/s, or have I missunderstod how the extensions work?

**Comment by kropp** [comment]
Correct.
My point is that after update the extension will be disabled for everyone and will require additional approval by user, even if they don't mind new functionality.

File: gitlab.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/** @author Johannes Tegnér <johannes@jitesoft.com> */
+import 'whatwg-fetch';
+import 'core-js/library/modules/es7.object.values';
```
Comment (maximmig) [inline gitlab.js]: You don't need to import Object.values since you use Object.keys

Comment (Johannestegner) [reply gitlab.js]: Ah yes, forgot to remove that one! Thank you, fixed :)

File: manifest.json [hunk 2]
```diff
@@ -15,13 +15,23 @@
...
     }
   ],
   "web_accessible_resources": [
     "*.svg"
   ],
   "permissions": [
     "activeTab",
-    "*://*.github.com/*"
+    "*://*.github.com/*",
+    "*://*.gitlab.com/*"
```
Comment (maximmig) [inline manifest.json]: Consider adding 'and GitLab' to the manifest description

Comment (Johannestegner) [reply manifest.json]: Fixed! ;)

**Pull request merged**